docs/process/issue-triage.md

```diff
@@ -98,11 +98,14 @@ issues from time to time that isn't and won't be covered here.
 
 ## More Information Needed
 
-Periodically we should be doing a sweep of issues that are open and labeled
-`more-information-needed`. If the original poster has not responded within
-two weeks after the last question by an official maintainer, close the issue
-with a message saying that you're closing the issue due to inactivity but are
-happy to re-open if they can provide more details.
+If we cannot understand or reproduce the issue with the information given, then we label it `more-information-needed`.
+
+Periodically, we should be doing a sweep of issues that are open and labeled
+`more-information-needed`. 
+* If the original poster has not responded within
+two weeks after the last question by an official maintainer, the issue will be automatically closed by Probot.
```

Comment:
thoughts on being more specific and saying `14 days` instead of `2 weeks`?

Reply:
> after the last question by an official maintainer, the issue will be automatically closed by Probot
whoa where do we configure probot to auto-close after 14 days since last question? that's pretty cool

Reply:
[Config for probot](https://github.com/desktop/desktop/blob/master/.github/no-response.yml)